src/java.desktop/unix/native/libpipewire/include/spa/param/audio/raw.h

```diff
@@ -11,7 +11,15 @@ extern "C" {
 
 #include <stdint.h>
 
+#if !defined(AIX)
 #include <spa/utils/endian.h>
+#endif
```

Comment:
The ifdef conditional blocks could be merged if I am not mistaken?

Reply:
Yes, at first I kept it as it was in the original fix.
But after revisiting it, I think that it is belongs in the new `spa/utils/endian.h` file.

@MBaesken could you please recheck if it still works for you?

Reply:
> Yes, at first I kept it as it was in the original fix. But after revisiting it, I think that it is belongs in the new `spa/utils/endian.h` file.
> 
> @MBaesken could you please recheck if it still works for you?

This still works for me on AIX.